tests/e2e/specs/pacbio/pacbio_libraries_view.cy.js

```diff
@@ -86,14 +97,15 @@ describe('Pacbio Libraries view', () => {
     cy.get('#libraryForm').should('not.exist')
 
     //It should display updated values in table
-    cy.get('#volume').first().should('have.text', '2')
+    cy.get('#volume').first().should('have.text', '3')
+    cy.get('#available_volume').first().should('have.text', '3')
```

Comment:
Can we also check the available_volume has changed by checking its original value above - around line 79.